docs/user/filesystem.md

```diff
@@ -989,6 +989,10 @@ libpil4dfs intercepting summary for ops on DFS:
 ### Force pil4dfs related env set in child processes when calling execve and its variants
 Normally child processes inherit environmental variables from parent processes. In rare cases, e.g. scons, envs are striped off when calling execve(). It might be useful to force pil4dfs related env set in child processes by setting env "D_IL_ENFORCE_EXEC_ENV=1". This flag is 0 if not set.
 
+### Turn on compatible mode in libpil4dfs
+
+"D_IL_COMPATIBLE=1" turns on compatible mode. Kernel fd allocated by dfuse instead of fake fd will be returned to applications. This mode provides better compatibility with degraded performance in open, openat, and opendir, etc. Please start dfuse with "--disable-caching --disable-wb-cache" to disable caching before using compatible mode.
```

Comment:
--disable-caching is sufficient without --disable-wb-cache

Reply:
It would be safer to disable writeback cache too. I will revise the document to make it more clear. Thank you!

Please start dfuse with "--disable-caching --disable-wb-cache" to disable caching and writeback caching before using compatible mode.

Reply:
--disable-caching disables all caching including wb-cache. so adding that is redundant

Reply:
ah. I did not know it. Thank you very much! I will revise it.